src/task.c

```diff
@@ -1098,6 +1145,9 @@ task_handler (gpointer user_data)
       break;
     case TASK_COMPLETED:
     {
+      if (!app_data->stdin)
+        rstrnt_upload_logs (task, app_data, soup_session, app_data->cancellable);
```

Comment:
I suspect the placement of this upload_logs will prevent a final message to be written further down in this function at line 1182.